.platform.app.yaml

```diff
@@ -0,0 +1,114 @@
+    # .platform.app.yaml
+
+# The name of this application, which must be unique within a project.
+name: 'app'
+
+# The type key specifies the language and version for your application.
+type: 'php:7.4'
+
+# On PHP, there are multiple build flavors available. Pretty much everyone
+# except Drupal 7 users will want the composer flavor.
+build:
+  flavor: composer
+
+# The relationships of the application with services or other applications.
+# The left-hand side is the name of the relationship as it will be exposed
+# to the application in the PLATFORM_RELATIONSHIPS variable. The right-hand
+# side is in the form `<service name>:<endpoint name>`.
+relationships:
+    database: 'db:mysql'
+
+# The hooks that will be triggered when the package is deployed.
+hooks:
+    # Build hooks can modify the application files on disk but not access any services like databases.
+    # build: |
+    # Deploy hooks can access services but the file system is now read-only.
+    deploy: |
+        set -e
+        rsync -a init/modules/* modules/
+        rsync -a init/img/* img/
+        rsync -a init/config/* config/
+        rsync -a init/translations/* translations/
+        rsync -a init/resources-translations/* app/Resources/translations/
+        rsync -a init/app-config/* app/config/
+        rsync -a init/themes/* themes/
+        rsync -a init/admin-dev/import/* admin-dev/import/
+        rsync -a init/mails/* mails/
+        rm -rf var/cache/*
+        if [ ! -f var/writeable/installed.lock ]; then
+            cd install-dev && php index_cli.php --db_server=database.internal --db_name=main --db_user=user --email=$ADMIN_EMAIL --password=$ADMIN_PASSWORD --domain=$(echo $PLATFORM_ROUTES | base64 -d | jq 'keys[0]'|sed 's|https\?://||'|sed 's|/||'|sed 's|\"||g')
```

Comment:
Please don't hard code the DB credentials.  We keep trying to train users out of that. :smile: They can be grabbed with jq easily enough at this point.